docs/src/main/sphinx/functions/ai.md

```diff
@@ -39,14 +38,58 @@ Configure multiple catalogs to use the same functions with different LLM
 providers. In this case, the functions must be referenced using their
 fully qualified name, rather than relying on the SQL path.
 
-## Providers
+:::{list-table} AI functions configuration properties
+:widths: 40, 60
+:header-rows: 1
+
+* - Property name
+  - Description
+* - `ai.provider`
+  - Required name of the provider. Must be `anthropic` for using the
+    [Anthropic provider](ai-anthropic) or `openai` for [OpenAI](ai-openai) or
+    [Ollama](ai-ollama).
+* - `ai.anthropic.endpoint`
+  - URL for the Anthropic API endpoint. Defaults to `https://api.anthropic.com`.
+* - `ai.anthropic.api-key`
+  - API key value for Anthropic API access. Required with `ai.provider` set to
+    `anthropic`.
+* - `ai.openai.endpoint`
+  - URL for the OpenAI API or Ollama endpoint. Defaults to
+    `https://api.openai.com`. Set to the URL endpoint for Ollama when using
+    models via Ollama and add any string for the `ai.openai.api-key`.
+* - `ai.openai.api-key`
+  - API key value for OpenAI API access. Required with `ai.provider` set to
+    `openai`. Required and ignored with Ollama use.
```

Comment:
Let's move these to the provider sections. Then it's more clear when they are used.

Reply:
Done